### ISSUE TITLE
Notification emails with multiple entry fields is seemingly not rendering properly

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#102]
**Description**

I'm not exactly sure what's going on here, but I have a form with two entries fields, each pulling from a different section. When the form submits, all is well and the data is saved as expected. Using the default "All Form Values" variable in a notification template, one of the entries fields shows up blank where the title would be. 

The email:
![image](https://user-images.githubusercontent.com/3810939/93414233-e2992600-f86e-11ea-9c63-9d5d5e7b5380.png)

The CMS:
![image](https://user-images.githubusercontent.com/3810939/93414427-491e4400-f86f-11ea-98a8-caa2699e3fd4.png)

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.2.12
- Craft version: 3.5.10.1
- Multi-site: No

**Comment by engram-design** [comment]
Just to confirm the "Product" field is an entries field, not a products field (and a Commerce Product element)?

We did have a similar issue with a multi-site https://github.com/verbb/formie/issues/95, but that was pretty specific to multi-site issues.

Anything particularly notable about the State field - or the section that it's linked to? In particular, I'd be interested to see your section settings.

So the fact that it's actually outputting a "li" element means that it found a matching element, but that it's having issues outputting the title of the element.

**Comment by engram-design** [comment]
This might be it - does your section not have URLs? If so, that's a bug I can reproduce.

**Comment by Mosnar** [comment]
@engram-design You're exactly right! The product field is an Entry with a URL, the state field is an Entry without a URL.

**Comment by engram-design** [comment]
Just fixed for the next release.

To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.2.14](https://github.com/verbb/formie/releases/tag/1.2.14)
